integration/docker-compose.yml

```diff
@@ -8,13 +8,19 @@ services:
       dockerfile: "Dockerfile"
     entrypoint: ""
     working_dir: "/src"
-    command: ["./integration/wait-for-it.sh", "mysql:3306", "--timeout=30", "--", "pytest", "storage_service", "integration"]
+    command: ["./integration/wait-for-it.sh", "mysql:3306", "--timeout=30", "--", "pytest", "integration"]
```

Comment:
> I've run the integration tests ~10-15 times now and haven't seen the intermittent failure you reported there. Have you seen it more than once, or just in the run you linked to?

Sorry for the confusion @tw4l. I removed `storage_service` here to avoid the issue on purpose. We need it back since this is the only way we have to run the SS tests under MySQL at the moment.

Reply:
ah okay, good to know!